conf/app.routes

```diff
@@ -20,4 +20,4 @@ GET         /session-timeout                        controllers.timeout.SessionT
 GET         /sign-out                               controllers.SignOutController.signOut
 
 #Sign In Routes
-GET         /sign-in                                controllers.SignInController.signIn
\ No newline at end of file
+GET         /sign-in                                controllers.SignInController.signIn
```

Comment:
We need a route adding in here that will allow BTA to call and retrieve the partial. This route will need Component Tests.